src/hotspot/share/classfile/bytecodeAssembler.hpp

```diff
@@ -179,37 +179,20 @@ class BytecodeAssembler : StackObj {
   void append(u2 imm_u2);
   void append(u4 imm_u4);
 
-  void xload(u4 index, u1 quick, u1 twobyte);
-
- public:
-  BytecodeAssembler(BytecodeBuffer* buffer, BytecodeConstantPool* cp)
-    : _code(buffer), _cp(cp) {}
-
-  void aload(u4 index);
-  void areturn();
   void athrow();
-  void checkcast(Symbol* sym);
-  void dload(u4 index);
-  void dreturn();
   void dup();
-  void fload(u4 index);
-  void freturn();
-  void iload(u4 index);
-  void invokespecial(Method* method);
   void invokespecial(Symbol* cls, Symbol* name, Symbol* sig);
-  void invokevirtual(Method* method);
-  void invokevirtual(Symbol* cls, Symbol* name, Symbol* sig);
-  void ireturn();
   void ldc(u1 index);
   void ldc_w(u2 index);
-  void lload(u4 index);
-  void lreturn();
   void _new(Symbol* sym);
-  void _return();
-
   void load_string(Symbol* sym);
-  void load(BasicType bt, u4 index);
-  void _return(BasicType bt);
+
+ public:
+  BytecodeAssembler(BytecodeBuffer* buffer, BytecodeConstantPool* cp)
+    : _code(buffer), _cp(cp) {}
+
+  static int assemble_method_error(
+    BytecodeConstantPool* cp, BytecodeBuffer* buffer, Symbol* errorName, Symbol* message);
```

Comment:
I think the more usual way to format the multi-line parameter list is
```
static int assemble_method_error(BytecodeConstantPool* cp,
(line up under prev param) --->     BytecodeBuffer* buffer,
(line up under prev param) --->     Symbol* errorName,
(line up under prev param) --->     Symbol* message);
```

Reply:
I fixed this.  This code was copied/moved from the other file with the odd formatting.